packages/block-library/src/social-link/README.md

```diff
@@ -0,0 +1,21 @@
+# Social Icon block
+
+The Social Icon is a foundational block that displays various icons linking to different social profiles or sites. Each social service is registered as a variation of this block.
+
+
+## Adding new social icons
+
+The core maintains certain standards for social icons. To add a new one to the existing variations, it must be a well-established, popular, and mainstream service.
+
+When considering establishment and popularity, several factors should be evaluated, such as:
+
+* Is the service popular enough for core developers to have heard of it before? Is it "mainstream?"
+* How old is the service?
+* Does it have a well-established Wikipedia article? (Seriously.)
+* Has anyone written a WordPress plugin that already adds it as a block variation? Do these plugins have any noticeable adoption or traction that would indicate usage and demand?
+* Is this social service frequently requested?
+
+
+## Adding custom social icons
+
+Starting from WordPress 6.9, it's possible to add custom social icons to your site. Here's an example plugin: https://github.com/wptrainingteam/devblog-custom-social-icons.
```

Comment:
We could include the dev block post when it's published.

Reply:
For now I would[ link to the PR](https://github.com/WordPress/gutenberg/pull/70261) that makes it possible in Gutenberg. And once the dev blog post is published add it to the readme.